health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/PopInbox.js

```
@@ -150,7 +155,7 @@
 
 
   useEffect(() => {
-    if (processData && processData.some((instance) => instance.action === "APPROVE_CENSUS_DATA")) {
+    if (processData && processData.some((instance) => instance.action === config?.disableInstanceAction)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)